recipes/scdoc/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class ScdocInstallerConan(ConanFile):
+    name = "scdoc"
+    description = "scdoc is a simple man page generator for POSIX systems written in C99."
+    topics = ("scdoc", "documentation")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://git.sr.ht/~sircmpwn/scdoc"
+    license = "Unknown"
+    settings = "os", "arch", "compiler", "build_type"
+    _autotools = None
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def build_requirements(self):
+        self.build_requires("make/[>=4.3.0]")
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def package_id(self):
+        del self.info.settings.compiler
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True,
+                  destination=self._source_subfolder)
+
+    @staticmethod
+    def _chmod_plus_x(filename):
+        if os.name == "posix":
+            os.chmod(filename, os.stat(filename).st_mode | 0o111)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self)
+        return self._autotools
+
+    def build(self):
+        autotools = self._configure_autotools()
+        with tools.chdir(os.path.join(self.source_folder, self._source_subfolder)):
+            autotools.make()
+            autotools.make(target="check")
+
+    def package(self):
+        autotools = self._configure_autotools()
+        with tools.chdir(os.path.join(self.source_folder, self._source_subfolder)):
```

Comment:
```suggestion
        with tools.chdir(self._source_subfolder):
```

Without `no_copy_source`

Reply:
corrected